From 4d773a5ce91ff1cdb68fc5f7c2be3be0041638ee Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonas=20Dre=C3=9Fler?= <verdre@v0yd.nl>
Date: Sun, 16 Feb 2020 21:56:27 +0100
Subject: [PATCH] appDisplay: Add back background color for folder icons

During this cycle the background color of the folder icons was removed
(probably fallout from the theme refactor), this was not intended
design-wise, so add it back.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/1011
---
 data/theme/gnome-shell-sass/widgets/_app-grid.scss | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/theme/gnome-shell-sass/widgets/_app-grid.scss b/data/theme/gnome-shell-sass/widgets/_app-grid.scss
index b23bea981b..1d2e135b44 100644
--- a/data/theme/gnome-shell-sass/widgets/_app-grid.scss
+++ b/data/theme/gnome-shell-sass/widgets/_app-grid.scss
@@ -35,9 +35,9 @@ $app_grid_fg_color: #fff;
 }
 
 /* App Folders */
-.app-folder {
-  .overview-icon {
-  }
+.app-well-app.app-folder {
+  background-color: transparentize($osd_bg_color, 0.8);
+  border-radius: $base_border_radius + 4px; // same as %icon_tile
 }
 
 // expanded folder
-- 
2.24.1

